migrations/versions/7883d80258fb_initial.py

```diff
@@ -0,0 +1,160 @@
+"""initial
```

Comment:
I think this is auto-generated, but if it isn't we need a license header.

Reply:
It is autogenerated-- I think we discussed it in the previous iteration of this PR.